### ISSUE TITLE
Picker isn't navigable with voice command tools

### ISSUE THREAD

**Issue by bessgreen** [facebook/react-native#31351]
## Description
Pickers are not fully navigable using voice command software:
     - iOS (VoiceOver): Pickers can be opened, but users are unable to scroll through the options
     - Android: Pickers cannot be opened

## React Native version:
0.63.4

### iOS: 
1. With Voice Control enabled, select the picker using spoken command
2. When the picker is opened, try to scroll through the options by saying "scroll up", "scroll down", "move down", "move up"
3. Note that the picker cannot scroll through the picker options

### Android:
1. With the device's native voice command tool enabled (e.g. Voice Access, Voice Assistant), try to select & open the picker using spoken command.
2. Note that the picker cannot be opened.

## Expected Results
Pickers should be fully navigable using the device's voice command tool. This includes the ability to:
     - Select, open, and close pickers
     - Scroll through the options in the picker
     - Select an option in the picker

**Comment by rectified95** [comment]
The `Picker` core module has been deprecated (mentioned here - https://reactnative.dev/docs/picker)
I suggest you use the community module implementation, instead:
https://github.com/react-native-picker/picker
